packages/keystone/lib/Keystone/index.js

```diff
@@ -75,6 +74,7 @@ module.exports = class Keystone {
     this.eventHandlers = { onConnect };
     this.registeredTypes = new Set();
     this._schemaNames = schemaNames;
+    this._expressOptions = Object.entries(expressOptions);
```

Comment:
```suggestion
```

Reply:
We're not accessing these options elsewhere so there is no need to set this property?